app/scripts/components/common/catalog/controls/hooks/use-catalog-view.ts

```diff
@@ -17,6 +25,44 @@ export function useCatalogView() {
   return {
     search,
     taxonomies,
-    onAction
+    onAction,
   };
-}
+} 
+// @NOTE: A hook using qs-state for query parameter management for cross-page navigation
+// Details: https://github.com/NASA-IMPACT/veda-ui/pull/1021
+export function useCatalogViewQS(): UseCatalogViewResult {
```

Comment:
⛏️ : What do you think about changing the name to `useFiltersUrlManagement` because this is mostly tied to the filters instead of the greater catalog view. We can probably change the interface to be `UseFiltersUrlManagementResults`... its long though..

Reply:
(This also required renaming of related variables to prefix by `Filter`, instead of `Catalog` !)